addons/common/fnc_isTurnedOut.sqf

```diff
@@ -6,75 +6,80 @@ Description:
     Checks whether a unit is turned out in a vehicle or not.
 
 Parameters:
-    _unit - Unit to check [Object]
+    _unit - Unit to check <OBJECT>
 
 Returns:
-    "true" for turned out or "false" for not turned out [Boolean]
+    "true" for turned out or "false" for not turned out <BOOL>
     Cargo in exposed vehicles are "turned out" if they lack a
     cargoSoundAttenuation matching index. Do NOT assume that because
     this function returns false that the unit is generally exposed.
 
-    For example, ATVs will return false, even though they are "turned
-    out".
-
 Examples:
     (begin example)
-    if ( [player] call CBA_fnc_isTurnedOut ) then
-    {
-        player sideChat "I am turned out!";
-    };
+        if (player call CBA_fnc_isTurnedOut) then {
+            player sideChat "I am turned out!";
+        };
     (end)
 
 Author:
-    Nou, courtesy of ACRE project.
+    shukari
 ---------------------------------------------------------------------------- */
-SCRIPT(isTurnedOut);
-
 params ["_unit"];
 
-private _turn = false;
-private _out = false;
+if (isNull objectParent _unit) exitWith {true};
+if (isTurnedOut _unit) exitWith {true};
+
+private _config = configFile >> "CfgVehicles" >> typeOf _vehicle;
```

Comment:
`_vehicle` is used before defined. You should test it.

Reply:
Forget to sync it with my testfile where i fixed it.
Thanks for spotting this.